### PR TITLE
Created empty projects

### PR DESCRIPTION
**Pull request by ParT1s**

### CODE REVIEW

**Comment by lalagvana** [comment]
Change AssemblyName and RootSpace with LT.DigitalOffice at the first place.
image

**Comment by lalagvana** [comment]
Also change netstandard from 2.0 to 2.1.

File: test/NewsService.Broker.UnitTests/NewsService.Broker.UnitTests.csproj [hunk 1]
```diff
@@ -0,0 +1,17 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>netcoreapp3.1</TargetFramework>
+
+    <IsPackable>false</IsPackable>
+
+    <AssemblyName>LT.DigitalOffice.NewsService.Broker.UnitTests</AssemblyName>
```
Comment (KostinP) [inline test/NewsService.Broker.UnitTests/NewsService.Broker.UnitTests.csproj:8]: RootNamespace?

File: LT.DigitalOffice.NewsService.sln [hunk 2]
```diff
@@ -0,0 +1,123 @@
+﻿
```
Comment (KostinP) [inline LT.DigitalOffice.NewsService.sln:1]: Empty line?

File: test/NewsService.Business.UnitTests/NewsService.Business.UnitTests.csproj [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+
+  <PropertyGroup>
+    <TargetFramework>netcoreapp3.1</TargetFramework>
+
+    <IsPackable>false</IsPackable>
+
+    <AssemblyName>LT.DigitalOffice.NewsService.Business.UnitTests</AssemblyName>
+
+    <RootNamespace>LT.DigitalOffice.NewsService.Business.UnitTests</RootNamespace>
+  </PropertyGroup>
```
Comment (djino97) [inline test/NewsService.Business.UnitTests/NewsService.Business.UnitTests.csproj:11]: Remove empty line, between tags, it does not looks very good

File: src/NewsService.Validation/NewsService.Validation.csproj [hunk 4]
```diff
@@ -0,0 +1,7 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+  <PropertyGroup>
```
Comment (djino97) [inline src/NewsService.Validation/NewsService.Validation.csproj:2]: add empty line

**Review by djino97 — APPROVED** [review]

**Review by lalagvana — APPROVED** [review]
Don't forget to resolve conversations if you've fixed the code

**Pull request merged**